charts/kubeai/templates/vllm-pod-monitor.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.metrics.prometheusOperator.vLLMPodMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ include "kubeai.fullname" . }}-vllm
+  labels:
+    {{- toYaml .Values.metrics.prometheusOperator.vLLMPodMonitor.labels | nindent 4 }}
```

Comment:
I would include common KubeAI labels (see: https://github.com/substratusai/kubeai/blob/883274cd151cf9c33758c4cacb59bbac57fafa0f/charts/kubeai/templates/role.yaml#L6) and append `.additionalLabels`.

Reply:
done